src/Polly.RateLimiting/RateLimiterResilienceStrategyBuilderExtensions.cs

```diff
@@ -0,0 +1,156 @@
+using System.Threading.RateLimiting;
+using Polly.Builder;
+using Polly.Utils;
+
+namespace Polly.RateLimiting;
```

Comment:
For discoverability, should we consider putting this in the same namespace as the non-Polly extension methods?

Reply:
I have no problem with that but maybe do the same with other extensions too?

Reply:
Move to just `Polly` namespace. (for timeout and retries too)